ReadMe.md

```diff
@@ -70,7 +70,7 @@ implementation 'io.github.quillraven.fleks:Fleks:1.0-RC3'
 #### Gradle (Kotlin)
 
 ```kotlin
-implementation("io.github.quillraven.fleks:Fleks:1.0-RC3")
+implementation("io.github.quillraven.fleks:Fleks:1.0-MPP")
```

Comment:
Is it MPP or KMM? If it is MPP then I will change the branch name ;)

Reply:
I just looked it up and the correct name is KMP for Kotlin MultiPlatform. KMM  stands for Kotlin Multiplatform Mobile. So would be only a special part of it;)

Reply:
Okay, I changed it to kmp. Please also change it then in the ReadMe and buil.gradle.kts version ;)